mlflow/models/evaluation/base.py

```diff
@@ -0,0 +1,530 @@
+from typing import Dict, Union
+import mlflow
+import hashlib
+import time
+import numpy as np
+import pandas as pd
+import json
+import os
+from mlflow.exceptions import MlflowException
+from mlflow.utils.file_utils import TempDir
+from mlflow.entities import Metric, RunTag
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
+from mlflow.utils import _get_fully_qualified_class_name
+from mlflow.utils.class_utils import _get_class_from_string
+import logging
+
+
+_logger = logging.getLogger(__name__)
+
+
+class EvaluationMetrics(dict):
+    """
+    Represent a dict of metrics.
+    """
+
+    pass
+
+
+class EvaluationArtifact:
+    """
+    Represent a artifact. Contains artifact uri and content.
+    """
+
+    def __init__(self, uri, content=None):
+        self._uri = uri
+        self._content = content
+
+    def _load_content_from_file(self, local_artifact_path):
```

Comment:
Can we document that this method should return loaded artifact content? In https://github.com/mlflow/mlflow/pull/5069/files#r759662089, we'll assign the loaded content to `self._content`.

Reply:
@WeichenXu123 This method should not be expected to assign to `self._content`. We should assign to `self._content` here: https://github.com/mlflow/mlflow/pull/5069/files#diff-7b34221ea99c68bbb283c3a4058bec08a43fcbf72392b8e9692853aa3ac6a4daR56. Sorry if my previous comment was unclear.